### ISSUE TITLE
Review of scSTEM repo

### ISSUE THREAD

**Issue by marouenbg** [alexQiSong/scSTEM#2]
Overall, it is very easy to install scSTEM using the first option and run it, it is very accessible.
I have a few minor comments:
- it would be nice to abstract a few steps by the command line, for example: the download of `lock` file could be done through a command like curl -O file.lock
- Also, you can specify the example path only once in the beginning as a variable for example path <- 'home/alex/sctem' and then refer to that variable for the remainder of the tutorial.
- You can also include the installation of the trajectory inference software as a dependency
- Finally, it would be nice to have an example button in the GUI that preloads some toy example data or even real world data.

Fantastic tool overall!

**Comment by marouenbg** [comment]
For the second round, I suggest the following:
- add releases to STEM, you can use GitHub release to add a microversion everytime you push new code/bug fix. For example, when the new GUI, this can be SCSTEM 0.1.1
- In README
install_folder = "/home/alex/scstem/"
renv::restore(project = install_folder, lockfile = paste0(install_folder,"renv.lock"), prompt = F)
- In README
add a section on how to update scTEM if a new release is developed. That would be through
1- `devtools::install_github("alexQiSong/scSTEM", upgrade = "always")` then 2-restart Rstudio.
- In GUI, the infer trajectory button could use a more expressive icon than a folder, I am not sure how shiny works, but if this is possible you can use icons from https://fontawesome.com/

Great job developing this phenomenal tool @alexQiSong !

**Comment by alexQiSong** [comment]
Thanks for the comments! Sorry to get back to this a bit late.

Thank you,
Alex
------
(Alex) Qi Song, Ph.D.
Postdoctoral Associate
Computational Biology Department
School of Computer Science
Carnegie Mellon University
Pittsburgh, PA 15213
